src/remote-config/remote-config.ts

```diff
@@ -333,6 +323,28 @@ class ServerTemplateImpl implements ServerTemplate {
       });
   }
 
+  /**
+   * Takes in either a {@link ServerTemplateData} or a JSON string
```

Comment:
Nit: we can now just reference ServerTemplateDataType

Reply:
Done 😀